FitGoal/FitGoal/Singup&Singin/CustomUIViews/AuthenticationFormView.swift

```diff
@@ -63,6 +69,31 @@ class AuthenticationFormView: UIStackView, UITextFieldDelegate {
         return true
     }
     
+    func textFieldDidBeginEditing(_ textField: UITextField) {
+        if let textField = textField as? CustomTextField {
+            if !textField.textFieldMessageLabel.isHidden {
+                textField.textFieldMessageLabel.isHidden = true
+            }
+        }
+    }
+    
+    func textFieldDidEndEditing(_ textField: UITextField) {
+        if let textField = textField as? CustomTextField {
+            guard let text = textField.text else { fatalError() }
```

Comment:
Avoid the fatal error here. You can ignore the optional value just writing `let text = textField.text ?? ""`